packages/service/src/launcher.ts

```diff
@@ -0,0 +1,112 @@
+import { rmdirSync } from 'node:fs'
+import logger from '@wdio/logger'
+import { SevereServiceError } from 'webdriverio'
+import type { Capabilities, Options } from '@wdio/types'
+import type { ClassOptions } from 'webdriver-image-comparison'
+import { BaseClass } from 'webdriver-image-comparison'
+import {
+    createStorybookCapabilities,
+    createTestFiles,
+    getArgvValue,
+    isCucumberFramework,
+    isStorybookMode,
+    parseSkipStories,
+    scanStorybook,
+} from './storybook.utils.js'
+import { CLIP_SELECTOR, NUM_SHARDS, V6_CLIP_SELECTOR } from './constants.js'
+
+const log = logger('@wdio/visual-service')
+
+export default class VisualLauncher extends BaseClass  {
+    #options: ClassOptions
+
+    constructor(options: ClassOptions) {
+        super(options)
+        this.#options = options
+    }
+
+    async onPrepare (
+        config: Options.Testrunner,
+        capabilities: Capabilities.RemoteCapabilities,
+        // For testing purposes
+        isStorybookModeFunc = isStorybookMode,
+        isCucumberFrameworkFunc = isCucumberFramework,
+        scanStorybookFunc = scanStorybook,
+        getArgvValueFunc = getArgvValue,
+        parseSkipStoriesFunc = parseSkipStories,
+        createTestFilesFunc = createTestFiles,
+        createStorybookCapabilitiesFunc = createStorybookCapabilities,
```

Comment:
Since all of these helper functions come from a different module, it is cleaner to just mock out the module directly. This approach should only be used as last straw when the function is part of the same module of class or function that you like to test

Reply:
Fixed it, thanks!